rpms.lock.tmp.yaml

```diff
@@ -0,0 +1,7794 @@
+---
```

Comment:
This file fails our big-files check. It should not appear at all.
Opened https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1758104213179319